docs/includes/es5-support.md

```diff
@@ -0,0 +1,4 @@
+For some versions of Office and Windows, the JavaScript engine in which add-ins run is provided by Internet Explorer. (For details of which versions, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).) The Internet Explorer engine does not support versions of JavaScript later than ES5. This means that without special handling the JavaScript files that your add-in serves cannot use syntax, types, or methods that were added to the language after ES5. This does not mean that you must *write* in ES5 syntax. You have two other options:
```

Comment:
Consider adding a comma after "special handling" to help readers parse out the conditional. 
```suggestion
For some versions of Office and Windows, the JavaScript engine in which add-ins run is provided by Internet Explorer. (For details of which versions, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).) The Internet Explorer engine does not support versions of JavaScript later than ES5. This means that without special handling, the JavaScript files that your add-in serves cannot use syntax, types, or methods that were added to the language after ES5. This does not mean that you must *write* in ES5 syntax. You have two other options:
```